src/frontend/staff-portal/proxy.conf.json

```diff
@@ -1,6 +1,6 @@
 {
   "/api": {
-    "target": "https://localhost:7136",
+    "target": "http://localhost:5005",
```

Comment:
Staff portal changed the proxy setting ? Just to make sure you are intentionally do this.

Reply:
this is ok since its only used when running on local host.  7136 is for when you run the staff-api from visual studio typically, 5005 is for running it from docker-compose up

Reply:
Yes, this is intentional. With this setting, I can run "docker-compose up" and run everything in docker except the portal and have the staff-portal just work against the staff-api running in docker.